boudicca.base/publisher-event-html/src/main/resources/templates/events/event.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="event">
         <div class="event-image {{#if event.category}}event-image-{{event.category}}{{/if}}">
             {{#if event.pictureUuid}}
-                <img src="/picture?uuid={{event.pictureUuid}}" style="height:100px;max-height:250px;max-width:300px;" />
+                <img src="/picture?uuid={{event.pictureUuid}}" style="height:100px;max-height:250px;max-width:300px;" alt="{{event.name}} bild" aria-label="Event Bild" aria-labelledby="image" />
```

Comment:
we have a new defined property called pictureAlt described here: https://github.com/boudicca-events/boudicca.events/blob/main/SEMANTIC_CONVENTIONS.md#L47

so we should try to actually use it

so please have a look if this property exists for an event, and if so you use that. if it does not exist use the text you are using right now as a fallback

Reply:
will check it out and use it.

Reply:
done